docker.go

```diff
@@ -515,7 +515,7 @@ func getEnviron(noHome bool) (result []string) {
 
 		switch varName {
 		case
-			"_", "PWD", "OLDPWD", "TMPDIR",
+			"_", "PWD", "PS1", "OLDPWD", "TMPDIR",
```

Comment:
When HOME is not mounted, shouldn't we refrain from exporting HOME here?  HOME on Mac is /Users/xxx, which won't exist in the Linux container.

Maybe we should map even less variables?  Right now, `tgf shell` on my mac always results in a garbled prompt, and I suspect it's probably because of PROMPT being mapped.  I use a fancy prompt on my host: https://starship.rs/ and its config is apparently not fully available in the container, even with HOME mounted.

Reply:
That's why I added PS1 in that list. Those are the variables that are not provided to the containers. HOME is already on that list since the beginning.

Reply:
My mistake: I thought these were variables we were mapping.  I tested this version and removing PS1 from the environment indeed fixes the `tgf shell` prompt for me too.